qiskit_experiments/calibration/backend_calibrations.py

```diff
@@ -0,0 +1,108 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Store and manage the results of a calibration experiments in the context of a backend."""
+
+from datetime import datetime
+from typing import List
+import copy
+
+from qiskit.providers.backend import BackendV1 as Backend
+from qiskit.circuit import Parameter
+from qiskit_experiments.calibration.calibrations_manager import CalibrationsManager, ParameterKey
+
+
+class BackendCalibrationsManager(CalibrationsManager):
+    """
+    A Calibrations class to enable a seamless interplay with backend objects.
+    This class enables users to export their calibrations into a backend object.
+    Additionally, it creates frequency parameters for qubits and readout resonators.
+    The parameters are named `qubit_freq_est` and `meas_freq_est` to be consistent
+    with the naming in backend.defaults(). These two parameters are not attached to
+    any schedule.
+    """
+
+    def __init__(self, backend: Backend):
+        """Setup an instance to manage the calibrations of a backend."""
+        super().__init__(backend.configuration().control_channels)
+
+        # Use the same naming convention as in backend.defaults()
+        self.qubit_freq = Parameter("qubit_lo_freq")
+        self.meas_freq = Parameter("meas_lo_freq")
+        self._register_parameter(self.qubit_freq)
+        self._register_parameter(self.meas_freq)
+
+        self._qubits = set(range(backend.configuration().n_qubits))
+        self._backend = backend
+
+    def get_frequencies(
+        self,
+        meas_freq: bool,
```

Comment:
Fair point. I added two new user-facing functions as you suggested. To avoid code duplication I kept the `meas_freq` argument but it is no longer in a user-facing method. See here: d19dfe8